docs/contribution_guide/fairlearn_repository_structure.rst

```diff
@@ -0,0 +1,29 @@
+Fairlearn Repository Overview
+-----------------------------
+
+Everything in the Fairlearn repository falls under one of the following
+categories:
+
+* python source code in the
+  `fairlearn <https://github.com/fairlearn/fairlearn/tree/master/fairlearn>`_
+  directory
+* python test code in the
+  `test <https://github.com/fairlearn/fairlearn/tree/master/test>`_ directory
+* visualization source code in the
+  `visualization <https://github.com/fairlearn/fairlearn/tree/master/visualization>`_
+  directory
+* automation / DevOps / CI pipeline definitions for
+    * Azure DevOps in the
+      `devops <https://github.com/fairlearn/fairlearn/tree/master/devops>`_
+      directory
+    * GitHub Actions in the
+      `.github <https://github.com/fairlearn/fairlearn/tree/master/.github>`_
+      directory
+    * CircleCI in the
+      `.circleci <https://github.com/fairlearn/fairlearn/tree/master/.circleci>`_
+      directory
+* documentation definition and configuration in the
+  `docs <https://github.com/fairlearn/fairlearn/tree/master/docs>`_ directory
+* example notebooks in the
+  `examples <https://github.com/fairlearn/fairlearn/tree/master/examples>`_
+  directory
```

Comment:
Fix the list styling (see the comment on "Ways to contribute"). And, for consistency, start the highest level lists with a capital letter. Also Python the language is a proper noun, so capitalize unless it's on a command line :-).

Reply:
looks a bit funny still, as if there's extra space in between, but better than before for sure
![image](https://user-images.githubusercontent.com/10245648/81652102-0d1e3280-93e8-11ea-9940-144bc31dd549.png)